ci/run_tests.sh

```diff
@@ -46,6 +46,11 @@ do
     fi
 
     PYTEST_CMD="pytest -m \"$TYPE_PATTERN$PATTERN\" -n $NUM_JOBS -s --strict --durations=10 --junitxml=test-data-$TYPE.xml $TEST_ARGS $COVERAGE pandas"
+    if [[ "$(uname)" == "Linux" ]]; then
+        # This is required for the clipboard to work (X virtual frambuffer implements the X server API, required by the clipboard tools, without running an actual X server)
+        DISPLAY=:99.0
```

Comment:
so u prob need to start xvfb much earlier and background it

Reply:
I'm using xvfb-run, which starts it for the command, and stops it when the command finishes. It's what xvfb's got in the azure servers. I guess something else can be installed, but I'll see what I can do.

May be time to try again to call the tests just once (not twice for single and multi). :)

Reply:
I think I’ve seen similar things for matplotlib headless tests.  IIRC they add a 1-second sleep after this for the thing to start